index.md

```diff
@@ -413,13 +414,17 @@ Taulukossa on kuvattu sanoman tietueiden käyttö.
 |Hakukriteeri|Tagi|Skeeman polku InfReqOpng/SchCrit/|Kuvaus|
 |:---|:---|:---|:---|
 |IBAN|\<IBAN\>|Acct/Id/Id||
+|\<MsgNmId\>|CstmrId/AuthrtyReq/Tp|"auth.001.001.01"|
```

Comment:
First column is missing in all four added rows.

Reply:
Fixed